src/silx/app/view/CustomPlotSelectionWindow.py

```diff
@@ -226,22 +243,51 @@ def clearAll(self):
         self._xDataset = None
 
         self._yParent.removeRows(0, self._yParent.rowCount())
-        
-        self._reset()
+
+        self._setXFile()
+        self._addYFile()
 
         self._plot1D.clear()
+        self._plot1D.setXAxisLabel("X")
+        self._plot1D.resetZoom()
 
         self._updateYCurvesWithDefaultX()
 
     def _reset(self):
         self._xDataset = None
 
-        self._addXFile("")
-        self._addYFile("")
+        self._setXFile()
+        self._addYFile()
 
         self._plot1D.clear()
         self._updateYCurvesWithDefaultX()
 
+    def _createTooltipAttributes(self, url):
+        attributeDict = {}
+
+        if url is not None:
+            attributeDict["#Title"] = "HDF5 Dataset"
+            attributeDict["Name"] = self._getTextSplit(url.data_path())
+            attributeDict["Path"] = url.data_path()
+            attributeDict["File name"] = self._getTextSplit(url.file_path())
+        else:
+            pass
+        return attributeDict
+    
+    def _getDefautToolTip(self, url):
+        attrs = self._createTooltipAttributes(url)
+        title = attrs.pop("#Title", None)
+        if len(attrs) > 0:
+            tooltip = _utils.htmlFromDict(attrs, title=title)
+        else:
+            tooltip = ""
+
+        return tooltip
+
+    def _getTextSplit(self, text):
+        textSplit = text.split("/")
+        return textSplit[len(textSplit) - 1]
```

Comment:
`len(textSplit)` is not needed, in python negative indices are starting from the end, so this does the same:

```suggestion
        return text.split("/")[-1]
```